documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -179,25 +179,29 @@ func buildClients(cfg *config) ([]writer, []reader) {
 		writers = append(writers, c)
 		readers = append(readers, c)
 	}
+	log.Info("Starting up ... ")
 	return writers, readers
 }
 
 func serve(addr string, writers []writer, readers []reader) error {
 	http.HandleFunc("/write", func(w http.ResponseWriter, r *http.Request) {
 		compressed, err := ioutil.ReadAll(r.Body)
 		if err != nil {
+			log.Errorln("Read Error: ", err)
```

Comment:
`log.Errorln()` automatically adds spaces between string arguments, so you won't need the extra space after the colon.